modules/authorization/policy-exemptions/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Policy Exemptions",
+  "summary": "With this module you can create policy exemptions across the management group, subscription or resource group scope.",
```

Comment:
Should the README.md be updated to "This module deploys Policy Exemptions at a Management Group, Subscription, or Resource Group scope" and then this file updated accordingly?